src/react/components/GenericForm.jsx

```diff
@@ -13,7 +14,15 @@ const { TextArea } = Input;
 
 // If the form is shown on popup the Select dropdown opens behind popup without this
 // FIXME: this will probably not work with modal popups (dropdown might be over the modal overlay)
-const zIndexValue = 999999;
+const zIndexValue = 99998;
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
This isn't the right place for this. Move it here: https://github.com/oskariorg/oskari-frontend/blob/master/src/react/components/Tooltip.jsx

Reply:
Though I'm not sure about this either. Does it create the global style just once or each time you use the `<Tooltip />`?

Reply:
Moved global styles to `portal.css` since Tooltip breaks down when adding child components.